### DISCUSSION TITLE
Anaokulu kalem sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#216] [Zeka Sorusu] [answered]
Anaokulunda 5 öğrencisi olan öğretmen, öğrencilerine elindeki farklı renkteki 5 kalemi dağıtmak istiyor. Kalemler, mavi, kırmızı, yeşil, mor ve turuncu renkteler. 5 öğrenciden 2’si kız (Fatma ve Ayşe), 3’ü erkek (Emre, Tolga ve Murat) öğrenciler. Öğretmen, her öğrencisine sevdiği renkteki bir kalemi vermek istiyor.
● Mavi kalemi bir erkek öğrenciye veriyor.
● Mor kalemi bir kız öğrenciye veriyor.
● Emre, mavi, yeşil ve turuncu renkleri sevmiyor.
● Ayşe, kırmızı ve mor renkleri sevmiyor.
● Murat turuncu kalemi alıyor.
Buna göre yeşil kalemi hangi öğrenci almıştır?

**Comment by adasue** [comment, marked as answer]
Önce elimizdeki bilgileri baştan okuyup not alarak ilerlememiz gerekiyor. Mavi kalemi bir erkek öğrenciye ve mor kalemi de bir kız öğrenciye veriyormuş şu an bundan bir çıkarım yapamayız. Emre mavi, yeşil ve turuncu renkleri sevmiyormuş. O zaman buradan mavi kalemi Tolga ya da Murat'ın aldığını çıkarabiliriz. Ayşe de kırmızı ve mor renkleri sevmiyormuş. Bilgilere göre mor kalemi kızlardan biri almış bu da Ayşe olamayacağına göre geriye Fatma kalıyor. Murat da turuncuyu almış. Mavi kalemi alan zaten Tolga ya da Murat'tan birisiydi eğer o turuncuyu aldıysa Tolga'ya da mavi kalmış oluyor. Ayşe kırmızı sevmediği için kırmızı Emre'ye veriliyor. **Geriye bir tek yeşil kalem kaldı onu da Ayşe alıyor.**
